google/cloud/security/common/data_access/cloudsql_dao.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides the data access object (DAO) for CloudSQL."""
+
+from MySQLdb import DataError
+from MySQLdb import IntegrityError
+from MySQLdb import InternalError
+from MySQLdb import NotSupportedError
+from MySQLdb import OperationalError
+from MySQLdb import ProgrammingError
+
+from google.cloud.security.common.data_access import project_dao
+from google.cloud.security.common.data_access import errors
+from google.cloud.security.common.data_access.sql_queries import select_data
+from google.cloud.security.common.util import log_util
+# pylint: disable=line-too-long
+from google.cloud.security.common.gcp_type import cloudsql_access_controls as csql_acls
+# pylint: enable=line-too-long
+
+LOGGER = log_util.get_logger(__name__)
+
+
+class CloudsqlDao(project_dao.ProjectDao):
+    """Data access object (DAO) for CloudSQL."""
+
+    def __init__(self):
+        super(CloudsqlDao, self).__init__()
+
+    def get_cloudsql_acls(self, resource_name, timestamp):
+        """Select the cloudsql acls for project from a snapshot table.
+
+        Args:
+            resource_name: String of the resource name.
+            timestamp: String of timestamp, formatted as YYYYMMDDTHHMMSSZ.
+
+        Returns:
+            List of cloudsql acls.
+
+        Raises:
+            MySQLError: An error with MySQL has occurred.
+        """
+        cloudsql_acls = {}
+        cnt = 0
+        try:
+            cloudsql_instances_sql = select_data.CLOUDSQL_INSTANCES.\
```

Comment:
There are a lot of backslashes for line continuation, which the style guide doesn't not recommend.
https://google.github.io/styleguide/pyguide.html?showone=Line_length#Line_length

Can we change these to use paranthesis?
```
cloudsql_instances_sql = (
    select_data.CLOUDSQL_INSTANCES.format(timestamp))